content/utilities/active-players/plugin.js

```diff
@@ -0,0 +1,144 @@
+// Active Players
+//
+// Get a list of which players have sent the most moves recently
+//
+// author: https://twitter.com/davidryan59 (dryan.eth)
+
+
+// -----------------------------
+// Constants and user editable parameters
+
+const GRAPH_API_URL = "https://api.thegraph.com/subgraphs/name/darkforest-eth/dark-forest-v06-round-5";
+// const GRAPH_API_URL = "https://api.thegraph.com/subgraphs/name/cha0sg0d/death-of-the-universe";
+
+const PLUGIN_NAME = "Active Players";
+const DEV_MODE = false;  // Put as true to highlight UI sections for debugging
+const ADDRESS_CHARS_TO_DISPLAY = 6;
+const OWNER_CHARS_TO_DISPLAY = 18;
+const MAX_RECORD_COUNT = 1000;  // GraphQL will retrieve up to 1000 objects in one go
+
+// -----------------------------
+
+
+const pg = df.getProcgenUtils();
```

Comment:
this function is removed in v0.6.5

Reply:
@davidryan59 added a PR on your branch that fixes the `pg.getProcgenUtils()` part. https://github.com/davidryan59/plugins/pull/2

Reply:
I've merged @cristobal's PR, this should be fixed now